modules/ocf_desktop/files/xsession/Xsession

```diff
@@ -118,3 +118,7 @@ sleep 8 && /usr/local/bin/fix-audio &
 
 # Source custom rc shared across desktops
 [ -f ~/remote/.desktoprc ] && . ~/remote/.desktoprc
+
+# Enable compositor
+xfconf-query -c xfwm4 -p /general/use_compositing -s false
+/usr/bin/compton --config $HOME/.config/compton/compton.conf --daemon
```

Comment:
not a huge fan of this pattern, it's pretty likely that this will break at some point in the future (typo in the config file, changes to compton when we upgrade desktops to a new version of debian, etc.) and nobody will notice since the output of this script just goes to `/dev/null` (and lack of compositor will take a while for anybody to notice)

Reply:
I suppose that's fair enough. We can pass all the necessary parameters like this instead (added some new stuff that seems to increase performance). 
```bash
compton  --daemon --config /dev/null --backend glx  \
--vsync opengl-swc --glx-no-stencil --paint-on-overlay \
--glx-swap-method 3 --glx-no-rebind-pixmap  \
--xrender-sync  --xrender-sync-fence  --unredir-if-possible \
--detect-client-opacity
```

Reply:
That doesn't solve the problem. Imagine e.g. that a new version stops accepting a command line argument. Is there any way we can design this to fail loudly?

Reply:
Passing an invalid arg makes `$? == 1`, and it's 0 otherwise. We could do a `notify-send` or write to ~/.xsession-errors, though no one checks that so it's not particularly loud.